log.go

```diff
@@ -61,6 +75,13 @@ func init() {
 	dcrlibwallet.UseLogger(dlwlLog)
 	page.UseLogger(pageLog)
 	load.UseLogger(log)
+	listeners.UseLogger(lstnersLog)
+	components.UseLogger(cmpntLog)
+	transaction.UseLogger(txntLog)
+	governance.UseLogger(govnLog)
+	walletPage.UseLogger(walpLog)
+	overview.UseLogger(ovewLog)
+	staking.UseLogger(stakLog)
```

Comment:
I don't think individual pages should have separate loggers, they should use the UI logger.

Reply:
Don't you think It'll be difficult to track down the particular source of an error if all the pages use UI logger?

Reply:
The individual page can/should add context, such as fmt.Errorf("wallets page: error doing so and so: %v", err). So, we don't have to create separate logs for each page. The UI logger should cover all UI-related errors, I believe.

Reply:
I can understand. But isn't it better not to rely on hardcoded page name in the error log. I'll refactor it but ultimately I believe it is more succinct if individual page have different logger.